README.md

```diff
@@ -7,18 +7,25 @@
 
 ## Installation
 
-The preferred way to install this extension is through [composer](http://getcomposer.org/download/).
+The preferred way to install this extension is through [composer](http://getcomposer.org/download/). yii2-froala editor depends on Froala PHP SDK
+To use and Install the editor, one must install PHP SDK also.
 
 Either run
 
 ```
 php composer.phar require --prefer-dist froala/yii2-froala-editor
+php composer.phar require --prefer-dist froala/wysiwyg-editor-php-sdk
+php composer.phar require --prefer-dist bower-asset/froala-wysiwyg-editor
```

Comment:
There is a composer package as well: https://packagist.org/packages/froala/wysiwyg-editor.